user-guide/Troubleshooting/Procedures/Troubleshooting_SAML_Issues.md

```diff
@@ -0,0 +1,112 @@
+
+DataMiner may face difficulties connecting to the identity provider due to various reasons:
+
+- An incorrect ipMetadata URL in the *Dataminer.xml* file (when connecting through a URL)
+
+- Improperly formatted *ipMetadata.xml* file (when using a local file)
+
+  Resolution:
+
+  - Ensure that the *ipMetadata.xml* file is correctly formatted, using tools like [Notepad++](https://notepad-plus-plus.org/downloads/) with an XML plugin. Missing `<?xml version="1.0" encoding="utf-8"?>` headers often lead to namespace issues.
+
+  - Be cautious with XML syntax restrictions, especially the use of "&," which should be replaced with `&amp;`.
+
+#### Using SAML together with a proxy
+
+When using SAML with a proxy, make these configurations to avoid authentication issues:
+
+- For 32-bit processes, execute the following command: `netsh winhttp set proxy proxy-server="<ip>:<port>" bypass-list="localhost"`
+
+- In *SLnet.exe.config*, add the following input under `<configuration>.<System.net>`:
+
+  ```xml
+  <defaultProxy>
+    <proxy
+     proxyaddress=[proxy address]
+     bypassonlocal="true"
+   />
+  ```
+
+  Replace placeholders with the correct IP addresses, ports, and addresses.
+
+### Missing or incorrect attribute statements
+
+The SAML attribute names must always match those in the IDP and the *DataMiner.xml* file.
+
+1. Go to the *C:\Skyline DataMiner* folder and open the *DataMiner.xml* file.
+
+1. Verify that the `<ExternalAuthentication>` tag is configured correctly, and the tag name for your attribute matches the definition.
+
+   For example, if the claim name is "email," your *Dataminer.xml* emailClaim tag should be `<EmailClaim>email</EmailClaim>`.
+
+1. If issues persist, use the *SAML-tracer* tool, a browser extension that captures SAML traffic.
+
+   1. In the Chrome web browser, add the [SAML-tracer extension](https://chrome.google.com/webstore/detail/saml-tracer/mpdajninpobndbfcldcmbpnnbhibjmch).
+
+   1. Go to `http(s)://[DMA name]/monitoring/`.
+
+   1. Click any received SAML response marked with an orange bubble labeled "SAML". Select this from the list and click *SAML* in the bottom window. Compare attribute names in the response to those in the IDP and the *Dataminer.xml* file.
+
+### Other frequent issues
+
+Here are some other common SAML-related issues:
+
+- **Duplicate users**: Ensure there are no duplicate users, and each email address in the *security.xml* file is unique.
+
+- **Groups in DataMiner**: Verify that groups exist in DataMiner before attempting to add users to them. This is necessary for setups using a separate claim for groups or a default group.
+
+- **Username format**:
+
+  - Ensure that the username is always an email address.
+
+  - In Azure setups, missing email field values can cause problems, even though the default username is an email.
+
+  - For other IDPs, configure DataMiner to use the correct claim, as outlined in the PreferredEmailClaim documentation.
```

Comment:
I'm referring to the Note under Azure B2C, I forgot that this wasn't a separate section because it is frequently needed for non standard setups (Azure AD)

DataMiner frequently can't find the claim that the IDP wants to use as username, the <PreferredEmailClaim> is a way to point DataMiner to the right one. this should perhaps be moved on the main page to avoid confusion and to point to it

Reply:
And where can I find the "PreferredEmailClaim" documentation? Could you comment with a link to that documentation? I would like to add a cross-reference.